test/e2e/framework/util.go

```diff
@@ -2615,6 +2616,55 @@ func NodeHasTaint(c clientset.Interface, nodeName string, taint *v1.Taint) (bool
 	return true, nil
 }
 
+//AddOrUpdateAvoidPodOnNode add avoidPods annotations to node, will override if it exists
+func AddOrUpdateAvoidPodOnNode(c clientset.Interface, nodeName string, avoidPods v1.AvoidPods) {
+	err := wait.PollImmediate(Poll, SingleCallTimeout, func() (bool, error) {
+		node, err := c.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
```

Comment:
@k82cn Thanks, will update all to use CoreV1() per comment here: https://github.com/kubernetes/kubernetes/pull/42890#discussion_r105520490